File: array abstracto/array.java
import java.util.Scanner;

public class Botella {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String[] botella = {"llena", "vacía"};

        System.out.println("¿Cómo quieres tu botella?");
        System.out.print("¿Llena o vacía?: ");
        String respuesta = scanner.nextLine().toLowerCase();

        if (respuesta.equals("llena")) {
            System.out.println(botella[0]);
        } else {
            System.out.println(botella[1]);
        }

        System.out.println("Gracias por tu compra");

        scanner.close();
    }
}

File: array abstracto/array.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
}); // Interfaz para leer entrada del usuario

console.log("¿Cómo quieres tu botella?");

rl.question("¿Llena o vacía?: ", (respuesta) => {
  const botella = ["llena", "vacía"];

  if (respuesta.toLowerCase() === "llena") {
    console.log(botella[0]);
  } else {
    console.log(botella[1]);
  }
  console.log("Gracias por tu compra");
  rl.close();
});

